src/data-table/data-table.styles.tsx

```diff
@@ -0,0 +1,156 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import { Checkbox } from "../checkbox";
+import { ArrowDownIcon, ArrowUpIcon } from "@lifesg/react-icons";
+// =============================================================================
+// STYLES
+// =============================================================================
+
+export const TableWrapper = styled.div`
+    width: 100%;
+    border: 2px solid ${Color.Neutral[6]};
+    border-radius: 0.5rem 0.5rem 0 0;
+    overflow: auto;
+`;
+
+export const Table = styled.table<{
+    $addMarginToFirstColumn?: boolean;
+    $addMarginToLastColumn?: boolean;
+}>`
```

Comment:
for these typings, pull them out into interfaces e.g. `TableStyleProps`

do add the STYLE INTERFACE header as well